### PR TITLE
Added ru-RU translation

### PR DESCRIPTION
**Pull request by MairwunNx**
image

### CODE REVIEW

**Comment by JasonStein** [comment]
Hi @MairwunNx, thx for the translation, can you rename your PR title to "Added ru-RU translation" for better consistency (with others PR)?

**Comment by MairwunNx** [comment]
Complete, i resolved merge conflict. Able to merge.

File: src/Notepads/Notepads.csproj [hunk 1]
```diff
@@ -356,6 +356,7 @@
     <PRIResource Include="Strings\tr-TR\Resources.resw" />
   </ItemGroup>
   <ItemGroup>
+    <PRIResource Include="Strings\ru-RU\Resources.resw" />
```
Comment (JasonStein) [inline src/Notepads/Notepads.csproj]: <ItemGroup>
    <PRIResource Include="Strings\ru-RU\Resources.resw" />
  </ItemGroup>
  <ItemGroup>
    <PRIResource Include="Strings\ja-JP\Resources.resw" />
  </ItemGroup>

File: src/Notepads/Notepads.csproj [hunk 2]
```diff
@@ -356,6 +356,7 @@
     <PRIResource Include="Strings\tr-TR\Resources.resw" />
   </ItemGroup>
   <ItemGroup>
```
Comment (JasonStein) [inline src/Notepads/Notepads.csproj]: <ItemGroup>
    <PRIResource Include="Strings\ru-RU\Resources.resw" />
  </ItemGroup>
  <ItemGroup>
    <PRIResource Include="Strings\ja-JP\Resources.resw" />
  </ItemGroup>

Comment (JasonStein) [reply src/Notepads/Notepads.csproj]: <ItemGroup>
    <PRIResource Include="Strings\ru-RU\Resources.resw" />
  </ItemGroup>
  <ItemGroup>
    <PRIResource Include="Strings\ja-JP\Resources.resw" />
  </ItemGroup>

Comment (JasonStein) [reply src/Notepads/Notepads.csproj]: <ItemGroup>
    <PRIResource Include="Strings\ru-RU\Resources.resw" />
  </ItemGroup>
  <ItemGroup>
    <PRIResource Include="Strings\ja-JP\Resources.resw" />
  </ItemGroup>

**Comment by JasonStein** [comment]
Hey, it should looks like these:

  <ItemGroup>
    <PRIResource Include="Strings\ru-RU\Resources.resw" />
  </ItemGroup>
  <ItemGroup>
    <PRIResource Include="Strings\ja-JP\Resources.resw" />
  </ItemGroup>

**Comment by MairwunNx** [comment]
@JasonStein sorry please, today I just drank one mug of coffee instead of two. For this such errors.

**Comment by JasonStein** [comment]
@MairwunNx  haha, no worries my friend.

**Comment by JasonStein** [comment]
Make sure you put your name here: https://github.com/JasonStein/Notepads/issues/33

**Comment by MairwunNx** [comment]
@JasonStein Thank you, pointed yourself there.

**Pull request merged**